scripts/bert/staticbert/index.rst

```diff
@@ -0,0 +1,166 @@
+Example Usage of Exporting Hybridizable BERT
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For exporting hybridizable BERT, we mean the BERT with fixed input embedding size and sequence length can be exported through
```

Comment:
Can we add two more sentences for the context? 

The BERTModel class is a subclass of Block, rather than HybridBlock. To support exporting BERT model to json format for deployment, we introduce the StaticBERT class.  Specifically, by exporting hybridizable BERT, we mean the BERT model with ...